packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -132,9 +143,11 @@ export default class ChildProcessWorker implements WorkerInterface {
       this._fakeStream.end();
       this._fakeStream = null;
     }
+
+    this._resolveExitPromise();
   }
 
-  onMessage(response: ParentMessage) {
+  private _onMessage(response: ParentMessage) {
```

Comment:
`onMessage` and `onExit` seem like worker implementation details to me, any reason why they should be public? `ChildProcessWorker` is not part of the `jest-worker` public interface either.

Reply:
I agree